snippets/column_defaults.yml

```diff
@@ -0,0 +1,19 @@
+- id: bbl
+  name: BBL
+  data_type: bbl
+  description: sample bbl description
+  example: '1016370141'
+- id: the_geom
+  name: the_geom
+  data_type: geometry
+  description: ''
```

Comment:
should the descriptions for these be the same since they seem to be placeholders and not really default values?